src/_gettsim/transfers/benefit_checks/benefit_checks.py

```diff
@@ -26,16 +26,16 @@ def wohngeld_vorrang_bg(
         See :func:`arbeitsl_geld_2_regelbedarf_m_bg`.
     arbeitsl_geld_2_eink_m_bg
         See :func:`arbeitsl_geld_2_eink_m_bg`.
-    wohngeld_nach_vermög_check_m_bg
-        See :func:`wohngeld_nach_vermög_check_m_bg`.
+    wohngeld_anspruchshöhe_m_bg
+        See :func:`wohngeld_anspruchshöhe_m_bg`.
 
     Returns
     -------
 
     """
     return (
         arbeitsl_geld_2_regelbedarf_m_bg
```

Comment:
I know it is a tiny thing, but given it is called `wohngeld_vorrang`, I have a strong opinion the check should go the other direction. 
```suggestion
        arbeitsl_geld_2_eink_m_bg + wohngeld_anspruchshöhe_m_bg >= arbeitsl_geld_2_regelbedarf_m_bg
```

Same for the two functions immediately below.